examples/notebooks/parameter-management.ipynb

```diff
@@ -0,0 +1,383 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Editing the default parameters\n",
+    "It is often much easier to define new parameters by editing exiting ones.\n",
+    "\n",
+    "We recommend not to alter the default parameters provided with the PyBaMM installation. Instead, you can pull all parameter set for\n",
+    "a given chemistry into the current directory:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Directory lithium-ion does not exist.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "if [[ ! -d \"lithium-ion\" ]]; then\n",
+    "echo \"Directory lithium-ion does not exist.\"\n",
+    "fi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "pybamm_edit_parameter lithium-ion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above commands will create a `lithium-ion` directory in the current directory, populated with copies of the default parameters, for editing:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "anodes\n",
+      "cathodes\n",
+      "cells\n",
+      "electrolytes\n",
+      "experiments\n",
+      "seis\n",
+      "separators\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "ls lithium-ion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As an example, let's create a new parameter file for the `cell` component, for the `lithium-ion` chemistry:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "mkdir lithium-ion/cells/my_new_param_set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create example parameter file\n",
+    "f = open(\"lithium-ion/cells/my_new_param_set/param_file.csv\", \"w+\")\n",
+    "f.write(\n",
+    "\"\"\"\n",
+    "Name [units],Value\n",
+    "a, 4\n",
+    "b, 5\n",
+    "c, 6\n",
+    "\"\"\"\n",
+    ")\n",
+    "f.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding a parameter directory\n",
+    "You can add a search location by modifying the `PARAMETER_PATH` list.\n",
+    "Let's add our new parameter directory to the list, in first position so that it is searched first:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['lithium-ion/cells/my_new_param_set',\n",
```

Comment:
Maybe we should add a brief comment about the first being a relative path and the others are absolute paths? I know it is quite basic but I was quite confused for a while :sweat_smile:

Reply:
As ever, It's basic only once you figure it out... I agree, this should be made clearer